br/pkg/lightning/restore/restore.go

```diff
@@ -1197,37 +1199,64 @@ func (rc *Controller) buildRunPeriodicActionAndCancelFunc(ctx context.Context, s
 						state = "preparing"
 					}
 
-					// since we can't accurately estimate the extra time cost by import after all writing are finished,
-					// so here we use estimatedWritingProgress * 0.8 + estimatedImportingProgress * 0.2 as the total
-					// progress.
+					// lightning restore is separated into restore engine and import engine, they are both parallelized
```

Comment:
It  is not quite accurate. `restore engine` includes `import engine` in implementation except index engine.

Reply:
here i mean `restoreEngine` which include encode/deliver and `importEngine` which includes write and ingest into tikv(for local mode) in table restore,

> except index engine.

yes, it's roughly divided, this part is not included.